packages/core/src/util.ts

```diff
@@ -12,14 +10,19 @@ const formatDate = (value: DiscordTimestamp): string | null => {
 	return padZeroes(`${value.getMonth() + 1}/${value.getDate()}/${value.getFullYear()}`);
 };
 
-export const handleTimestamp = (value: DiscordTimestamp): string | null => {
+const formatTime = (value: DiscordTimestamp): string | null => {
```

Comment:
Add an option for 24-hour time.

That'll then have to bubble all the way up to DiscordMessage so people can set it.

Reply:
```html
<discord-message profile="skyra" twenty-four>
	<discord-reply slot="reply" profile="skyra" command>Pong!</discord-reply>
	Took 100ms.
</discord-message>
```